.devcontainer/devcontainer.json

```diff
@@ -1,5 +1,5 @@
 {
- "image": "mosaicml/pytorch_vision:latest",
```

Comment:
@snarayan21 wanted to make sure this change was cool with you? We are moving away from the pytorch_vision images.